qa/L0_lifecycle/slow_load_repo/identity_slow_load/config.pbtxt

```diff
@@ -0,0 +1,49 @@
+# Copyright 2022, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions
+# are met:
+#  * Redistributions of source code must retain the above copyright
+#    notice, this list of conditions and the following disclaimer.
+#  * Redistributions in binary form must reproduce the above copyright
+#    notice, this list of conditions and the following disclaimer in the
+#    documentation and/or other materials provided with the distribution.
+#  * Neither the name of NVIDIA CORPORATION nor the names of its
+#    contributors may be used to endorse or promote products derived
+#    from this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS ``AS IS'' AND ANY
+# EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+# PURPOSE ARE DISCLAIMED.  IN NO EVENT SHALL THE COPYRIGHT OWNER OR
+# CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+# EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+# PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+# PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY
+# OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+# (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+name: "identity_slow_load"
+backend: "identity"
```

Comment:
Avoid using / extending identity backend if possible, can't observe the speed up with two Python model?

Reply:
I can't observe the speed up with two Python models:
```
I1103 18:30:44.668903 3610 model_lifecycle.cc:459] loading: python_slow_load_1:1
I1103 18:30:44.669170 3610 model_lifecycle.cc:459] loading: python_slow_load_2:1
I1103 18:30:45.940105 3610 python_be.cc:1856] TRITONBACKEND_ModelInstanceInitialize: python_slow_load_1 (GPU device 0)
I1103 18:30:56.127921 3610 python_be.cc:1856] TRITONBACKEND_ModelInstanceInitialize: python_slow_load_2 (GPU device 0)
I1103 18:30:56.127991 3610 model_lifecycle.cc:694] successfully loaded 'python_slow_load_1' version 1
I1103 18:31:06.266695 3610 model_lifecycle.cc:694] successfully loaded 'python_slow_load_2' version 1
```
(Each Python model will delay for 10 seconds during initialization)
I guess the Python is using the same interpreter for both models. If so, then the GIL will remove any parallelism which explains the result.

However, the speed up is observed with two Identity models:
```
I1103 18:31:58.640473 4141 model_lifecycle.cc:459] loading: identity_slow_load_2:1
I1103 18:31:58.640715 4141 model_lifecycle.cc:459] loading: identity_slow_load_1:1
...skip...
I1103 18:31:58.642488 4141 identity.cc:232] Creation delay is set to: 10
I1103 18:31:58.642509 4141 identity.cc:232] Creation delay is set to: 10
I1103 18:32:08.645463 4141 identity.cc:662] TRITONBACKEND_ModelInstanceInitialize: identity_slow_load_1 (GPU device 0)
I1103 18:32:08.646140 4141 model_lifecycle.cc:694] successfully loaded 'identity_slow_load_1' version 1
I1103 18:32:08.646241 4141 identity.cc:662] TRITONBACKEND_ModelInstanceInitialize: identity_slow_load_2 (GPU device 0)
I1103 18:32:08.646658 4141 model_lifecycle.cc:694] successfully loaded 'identity_slow_load_2' version 1
```

Reply:
@Tabrizian don't we use separate processes to work around the GIL?

Reply:
Since the L0_lifecycle already use one identity model, maybe we can stick with identity models instead?

Reply:
Yes, we do use separate processes to work around GIL. Are we sure that separate `TRITONBACKEND_ModelInstanceInitialize` are called for each Python backend model instance?

Reply:
> Since the L0_lifecycle already use one identity model, maybe we can stick with identity models instead?

Given the situation, we will have to use identity backend for now. If possible, I will avoid adding any more use of identity backend, it is used in some tests is a legacy usage (don't have Python backend back then). The reason is that C/C++ backend is not flexible for testing where the models often need to execute in certain way and all those configurations are stuff into single backend file, while Python backend model is more flexible and clear in the way that you can define different model in dedicated model file. What's more, since there is dedicated model file, it is easier to look into the file and understand how exactly the model operates